asllib/doc/ASLSemanticsReference.tex

```diff
@@ -235,13 +2024,13 @@ \section{SemanticsRule.Lit \label{sec:SemanticsRule.Lit}}
 \section{SemanticsRule.ELocalVar \label{sec:SemanticsRule.ELocalVar}}
 
   \subsection{Prose}
-  Evaluation of the expression \texttt{e} under environment \texttt{env} is
-  \texttt{(res,new\_env)} and all of the following apply:
+  Let \texttt{e} be an expression denoting a variable \texttt{x}.
+  Evaluation of \texttt{e} under environment \texttt{env} terminates normally with
+  \texttt{((v,g),env)} and all of the following apply:
```

Comment:
looking at the old prose, you have changed res into v, and new_env into env. have you changed the names in the code too?
please avoid changing names too much.
furthermore, as above, please could you use your maths macros in the prose description too, so we are sure we've got the same names and notations in the prose and the maths?
this request to use the macros applies to all the rest of the rules too please.

Reply:
I changed back. However, res and new_env are not part of the code - the code does not name them at all.
I'll use the name macros in all rules.